elements/lisk-cryptography/src/ed.ts

```diff
@@ -100,9 +85,13 @@ export const signMessageWithPrivateKey = (
 	message: string,
 	privateKey: Buffer,
 ): SignedMessageWithPrivateKey => {
-	const msgBytes = digestMessage(message);
 	const publicKey = getPublicKey(privateKey);
-	const signature = signDetached(msgBytes, privateKey);
+	const signature = signDataWithPrivateKey(
```

Comment:
I think to allow more generic message signing, we should allow to pass the tag in this function or expose `signDetached` and `verifyDetached` from the cryptography library